int64 blockNumber
int64 productNumber
string timeStamp
string station
string serialNumber
string blockHash
string previousHash
